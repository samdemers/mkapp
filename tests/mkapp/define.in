# Create modules, run them and redefine them as they are running. They should
# be shut down cleanly and replaced by the newly defined module.
define module1 echo "Error!";
run    module1;
define module1 echo "\n"
                    "\t\"Hello, world!\"\n"
                    "\t\\t\\\"Hello, world!\\\"\\n" "\n"
                    "\t"'\t"Hello, world!"\n';
listen module1;
run    module1;
