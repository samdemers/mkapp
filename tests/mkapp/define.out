
 	"Hello, world!"
 	\t\"Hello, world!\"\n 
 	\t"Hello, world!"\n
